gateway/src/main/java/com/hurence/webapiservice/historian/HistorianFields.java

```diff
@@ -32,6 +32,8 @@ private HistorianFields() {}
     public static String RESPONSE_CHUNKS = "chunks";
     public static String RESPONSE_METRICS = "metrics";
     public static String RESPONSE_TOTAL_FOUND = "total_hit";
+    public static String RESPONSE_TOTAL_NAMES_FOUND = "total_names";
+    public static String RESPONSE_TOTAL_METRICS_FOUND = "total_metrics";
```

Comment:
I would prefer those names. If you accept this you will have to modify your response file in tests. NAME and METRICS means the same thing, so i think this is ambiguous.
```suggestion
    public static String RESPONSE_TOTAL_METRICS_RETURNED = "total_returned";
    public static String RESPONSE_TOTAL_METRICS_FOUND = "total";
```

Reply:
yes your choice is better